addons/mail/models/models.py

```diff
@@ -201,3 +201,13 @@ def _notify_by_email_get_headers(self):
         return {
             'X-Odoo-Objects': "%s-%s" % (self._name, self.id),
         }
+
+    # ------------------------------------------------------------
+    # TOOLS
+    # ------------------------------------------------------------
+
+    def _get_html_link(self, title=None):
+        self.ensure_one()
+        return f"<a href=# data-oe-model={self._name} data-oe-id={self.id}>"\
+               f"{tools.html_escape(title or self.display_name)}"\
+               f"</a>"
```

Comment:
`tools.html_escape` is actually `markupsafe.escape` now. You can also do something like
```py
Markup("<a data-oe-model='%s' ...>%s</a>") % (self._name, ..., title or self.display_name)
```

Reply:
Well, at least html_escape doesn't trigger the ci/security, whereas markup does :D